### ISSUE TITLE
Crash when eating Terraqueous fruits

### ISSUE THREAD

**Issue by drantral** [ShetiPhian/Localizations-and-Issues#41]
Have yet to test out every fruit, but so far each one; oranges, pears, etc cause minecraft to crash. Here's a link to the crash log: [http://pastebin.com/CYH692jC](http://pastebin.com/CYH692jC)

**Comment by ShetiPhian** [comment]
Looks to be a conflict with another mods ASM.

Eating is working in the development environment.
The crash is also happening in a spot that has no reason to crash. The only variable is the instance of the food class; that cant be null or there wouldn't have been anything to eat in the first place. The other code on that line pulls info from my class, if that was the issue the crash would have happened in my class.

Someone is adding code to the start of FoodStats.addStats until I know who I can't see why the crash happens.
I thought it was AppleCore but it is adding to the Integer&Float one not the ItemFood&ItemStack one in the crash.

**Comment by ShetiPhian** [comment]
It is an incompatibility with Apple Core's ASM
I'm not too good at reading ASM code so I was unable to see what was causing the crash -_- 

Instead I did a few quick tests and it appeared to only happen when you ate the last item in the stack.
So I changed some code execution order (a.k.a. I moved one line), and it appears to be fixed.

Here is a test build for you:
http://www.mediafire.com/file/b7ccqnubxr4kzd3/terraqueous-1.11.0-1.3.7-rb1.jar
